src/components/Restoration/RestorationType.vue

```diff
@@ -119,10 +138,31 @@ export default class RestorationType extends Vue {
   @Watch('getRestoration.type', { immediate: true })
   private onRestorationType (val: RestorationTypes): void {
     this.selectRestorationType = val
-
-    // if Limited, set validity to true
-    // if Full then validity is set by RelationshipsPanel
-    if (this.isLimitedRestorationFiling) this.setRestorationTypeValid(true)
   }
 }
 </script>
+
+<style lang="scss" scoped>
+@import '@/assets/styles/theme.scss';
+.tooltip-text {
+  color: $gray7;
+}
+
+:deep() {
+  // Fix font of all the radio buttons.
+  .radio-button {
+    label {
+      font-weight: normal;
+      color: $gray7;
+    }
+  }
+
+  // Fix font of the relationships panel & center text with the checkboxes.
+  .v-input--checkbox .v-input__control .v-input__slot .v-label {
+    color: $gray7;
+    font-weight: normal;
+    align-items: stretch;
+    height: 1.3rem;
```

Comment:
This is a very weird value. How did you decide on it? Why is it not a multiple of 0.25rem? Or if it's a specific value then maybe it should be in px instead.